projects/ui/option/option.html

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="16"
+  height="16"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+  aria-hidden="true"
+  [ngClass]="{ visible: isSelected(), invisible: !isSelected() }"
+>
+  <path d="M20 6 9 17l-5-5" />
+</svg>
```

Comment:
To do it correctly we would have to implement a genetic Icon component, which would allow "plugging-in" any icon library users might possibly use with all the ensuing consequences..

I was thinking that is too much for this lib, as the idea of it was that user can install individual components to their codebase and rearrange things however they want to. So there's a (thin) line between basic customizations we want to support and more complex things that is better to give it up for grabs to the consumer..